src/main/kotlin/no/nav/familie/oppdrag/repository/SimuleringLagerTjenesteImpl.kt

```diff
@@ -0,0 +1,28 @@
+package no.nav.familie.oppdrag.repository
```

Comment:
Trenger man denne klassen? Eller er det tilsrekkelig for SimuleringTjenesteImpl å kalle direkt på SimuleringLagerRepository? 

Det ende som er denne trengs til er `@Transactional(propagation = Propagation.REQUIRES_NEW)`, men ser ikke noe behov for den?

Reply:
Ja, det trenger vi. Request må lagres i en separat transaksjon. Ellers lagres ikke request-XML i database om simulering feiler.